instrumentation/net/http/otelhttp/stats.go

```diff
@@ -0,0 +1,206 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Naming this file `stats.go` seems like a hold over from OpenCensus. Can we rename it to `metrics.go` or something more idiomatic of OTel and the terminology of the project?

Reply:
Renamed to `metrics.go`